ui/imports/shared/popups/send/controls/GasSelector.qml

```diff
@@ -16,18 +16,19 @@ Item {
 
     property var selectedAsset
     property string currentCurrency
```

Comment:
It's probably not needed. It's taken only to be returned back via callback function. Also the function can be simplified then to take only one parameter.